drivers/storage/portworx/component/csi.go

```diff
@@ -909,6 +919,10 @@ func getCSIStatefulSetSpec(
 	}
 	imagePullPolicy := pxutil.ImagePullPolicy(cluster)
 
+	sc := &v1.SecurityContext{
```

Comment:
@zoxpx let's create a ticket to remove this stateful set. This was needed for pre 1.13 k8s versions. We don't support that old version anymore.